exasol/analytics/udf/communication/socket_factory/fault_injection.py

```diff
@@ -64,7 +64,7 @@ def receive(self) -> bytes:
     def receive_multipart(self) -> List[abstract.Frame]:
         message = self._internal_socket.receive_multipart()
         converted_message = [Frame(frame) for frame in message]
-        return converted_message
+        return converted_message  # type: ignore
```

Comment:
Why do you need the type ignore comment here?

Reply:
I got 
* "List" is invariant -- see https://mypy.readthedocs.io/en/stable/common_issues.html#variance
* Consider using "Sequence" instead, which is covariant
* Perhaps you need a type annotation for "converted_message"? Suggestion: "list[abstract.Frame]"

But adding this type annotation seems to fix the problem.

Reply:
See next push.